### ISSUE TITLE
Not able to turn Settings Sync on after changing github password

### ISSUE THREAD

**Issue by robole** [microsoft/vscode#104816]
Version: 1.48.0 (system setup)
Commit: db40434f562994116e5b21c24015a2e40b2504e6
Date: 2020-08-13T07:50:42.600Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18362

Steps to Reproduce:

1. Change github password!
2. Open User Settings
3. Press "Turn on Settings Sync" button
4. Choose settings to sync
5. Select GitHub account (as per screenshot).
6. You will receive a 401 error  (as per screenshot), but there is no option to sign in again with GitHub, or choose another Github Account.

![sync](https://user-images.githubusercontent.com/3703647/90394642-9a4ae480-e0c5-11ea-90f9-4679c69b3bd6.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
@RMacfarlane assigning to you as this is related to auth and it seems user changed the password causing this?

**Comment by RMacfarlane** [comment]
Closing as a duplicate of https://github.com/microsoft/vscode/issues/104595

**Comment by robole** [comment]
Hi @RMacfarlane 

The other issue #104595 is different than this issue. I didnt want to switch my github account, which is the crux of the other issue. My issue is that I changed password in github, and was getting a 401 error in VS Code afterwards, with no way to resolve it.

If you think the same solution will solve both issues, that's fine.  But this is a different scenario and should be tested also to see if it is resolved.

**Comment by RMacfarlane** [comment]
Yeah, my thinking was that both issues could be solved by the same change, but I'll reopen this to make sure this scenario gets tested then. Thanks!

**Comment by TylerLeonhardt** [comment]
This also happens if you revoke the OAuth App permission
